addons/a11y/src/index.ts

```diff
@@ -56,7 +55,6 @@ export const withA11y: StoryWrapper = (getStory, context) => {
   return getStory(context);
 };
 
-channel.on(STORY_RENDERED, () => run(setup.element, setup.config, setup.options));
```

Comment:
Shouldn't we clear the page when the story changed?

The result might be incorrect..

Reply:
We still have an old state in the A11YPanel component.
But 
- if the tab is inactive, the old report is not displayed
- when tab becomes active, the old report is not rendered, I changed the behavior to display only the status button when it is running (on tab activation, a REQUEST event is sent, and status is running)
- if tab is active, on story change, a REQUEST event is sent, hiding the old report (see previous point) while running